src/prefect/deployments/base.py

```diff
@@ -236,6 +236,7 @@ def _format_deployment_for_saving_to_prefect_file(
                 schedule_config = deployment_schedule.schedule.model_dump()
 
             schedule_config["active"] = deployment_schedule.active
+            schedule_config["parameters"] = deployment_schedule.parameters
```

Comment:
i don't like this pseudo/shadow duplication of the schema here but I didn't want to over-complicate this PR